support/config/deployment.go

```diff
@@ -269,6 +269,8 @@ func (c *DeploymentConfig) setControlPlaneIsolation(hcp *hyperv1.HostedControlPl
 		})
 	}
 
+	c.Scheduling.Tolerations = append(c.Scheduling.Tolerations, hcp.Spec.Tolerations...)
+
```

Comment:
Can you please also add a unit test in support/config/deployment_test.go ?
You can take it from https://github.com/openshift/hypershift/pull/4337/files#diff-17ac58455ee34787c5ace14ebafe8b2e3ee34bb36ff47bc5dc73ea1b0835cc41

Reply:
I cherry picked that part of your PR in as a separate commit and set you as the author